lib/text/ttml_text_parser.js

```diff
@@ -188,12 +190,13 @@ shaka.text.TtmlTextParser = class {
    * @param {boolean} whitespaceTrim
    * @param {boolean} isNested
```

Comment:
The `isNested` property isn't really needed anymore, with this change. We can just check if `parentCueElement` is non-null, now.

Reply:
ok, I'll change it!

Reply:
Done!